pages/docs/community/styleguide/links.md

```diff
@@ -0,0 +1,35 @@
+# Link Style Guide 📃
+
+Links are an essential part of any written content, as they allow readers to quickly navigate to other related information. A well-designed link structure can greatly enhance the user experience, while a poorly designed one can be confusing and frustrating. In this style guide, we will outline the guidelines for designing links in text, assets, and external links.
+
+## Link Text 📝
+
+ The Link Text heading is all about how to make your links more effective and user-friendly. Instead of using generic phrases like "click here," you'll learn how to use descriptive and concise text that accurately reflects the content of the linked page. By following these guidelines, your links will be more informative and engaging for your readers.
+
+- Use descriptive and concise text that accurately reflects the content of the linked page. For example: **"Learn more about AsyncAPI"** instead of **"click here"**.
+- Avoid using general words such as "click here" or "read more."
+- Make sure the link text accurately describes the target of the link. For example: **"Check out the latest version of AsyncAPI"** instead of **"click here for more information"**.
+- For external links, it is recommended to indicate that it is an external link by using the text **external link**.
+
+## External Links 🌐
+
+Linking to external websites is a common practice, but it can be tricky to do it in a way that's user-friendly. This section will show you how to indicate that a link is external and ensure that it opens in a new tab. By following these guidelines, you'll make it easier for your readers to navigate between your content and external resources without losing their place.
+
+- Use target=**"_blank"** attribute to open external links in a new tab. This makes it easier for the user to return to the original page.
+- Ensure that the external links open in a new tab or window, and not in the same tab, so as to avoid losing the original page. For example, if you click on an external link, a new tab should open and the original page should still be accessible.
+
+## Link Structure 📊
+
+Having a consistent and clear link structure can greatly improve the readability and flow of your content. In this section, you'll learn how to use a consistent link structure throughout your content and avoid using too many links in one section. By doing so, you can make your content more organized and easier to read, while also avoiding distractions or confusion.
```

Comment:
```suggestion
Having a consistent and clear link structure can greatly improve the readability and flow of your content. Use a consistent link structure throughout your content and avoid using too many links in one section. By doing so, you can make your content more organized and easier to read, while also avoiding distractions or confusion.
```

The words `In this section, you'll learn how to` sound like this is a tutorial. Let's be succinct :D

Reply:
Okay Mam